src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
-);
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo todo={todo} key={todo.id} />
+      ))}
+    </section>
+  );
+};
```

Comment:
on my opninion, when you function immediately return component, better to write it in round braces, so you dont need return statement
```suggestion
export const TodoList: React.FC<Props> = ({ todos }) => (
  <section className="TodoList">
    {todos.map(todo => (
      <TodoInfo todo={todo} key={todo.id} />
    ))}
  </section>
);
```